Archive/Wiedijk100Theorems/BallotProblem.lean

```diff
@@ -72,6 +72,7 @@ def countedSequence (p q : ℕ) : Set (List ℤ) :=
   {l | l.count 1 = p ∧ l.count (-1) = q ∧ ∀ x ∈ l, x = (1 : ℤ) ∨ x = -1}
 #align ballot.counted_sequence Ballot.countedSequence
 
+open scoped List in
```

Comment:
Why are we doing this?

Reply:
I see, it's scoped notation for Perm now. I guess that's okay. It might make more sense to just open it in the entire file when needed, but that can be changed later if preferable.